migrations/0055_set_timestamps_trigger.sql

```diff
@@ -0,0 +1,10 @@
+-- Create the function to set timestamps with triggers
+
+CREATE FUNCTION set_timestamps()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW."createdAt" := COALESCE(NEW."createdAt", NOW());
```

Comment:
Why do we need to set this, if it is being defied in the table schema?

Reply:
I thought that a trigger like this already existed...but I think the `updatedAt` columns are only getting updated by TypeORM, rather than Postgres automatically updating the `updatedAt` column via trigger. The problem I was running into is I need the `updatedAt` column to be up to date for stats, and also, the stats queries don't use TypeORM's special helpers, but rather makes direct sql queries for updates...which means the stats queries weren't automatically updating the `updatedAt` values. The trigger handles that now.